h2o-algos/src/main/java/hex/StackedEnsembleModel.java

```diff
@@ -73,6 +74,7 @@ public StackedEnsembleModel(Key selfKey, StackedEnsembleParameters parms, Stacke
       deeplearning
     }
     public MetalearnerAlgorithm _metalearner_algorithm = MetalearnerAlgorithm.AUTO;
+    public String _metalearner_params = new String();
```

Comment:
FYI: Can be a map in the future for java api purposes

Reply:
I will have to think about this more. Not necessary for this PR, right?

Reply:
Nope, not necessary to address right now, just thinking about the future.